backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/attribute/impl/HUUniqueAttributesService.java

```diff
@@ -237,4 +240,36 @@ public void createOrUpdateHUUniqueAttribute(@NonNull final HuId huId)
 				.forEach(this::createOrUpdateHUUniqueAttribute);
 	}
 
+	public void validatePPOrderASI(@NonNull final I_PP_Order ppOrder)
```

Comment:
why not taking productId and asiId as params instead of the whole ppOrder?
In this way we would avoid having an unnecessary dependency from handlingunits/attributes to PP_Order ...